newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenuBar.js

```diff
@@ -69,21 +86,25 @@ const HomePageMenuBar = ({
 
   const buttons: {
     label: React.Node,
-    getIcon: (color: string) => React.Node,
+    getIcon: $PropertyType<HomePageMenuTab, 'getIcon'>,
```

Comment:
👀  what is this, should you type the function and import it instead?

Reply:
You're right, it will be clearer